swap_meet/item.py

```diff
@@ -1,2 +1,21 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category="", condition=0):
+        self.category = category
+        self.condition = condition
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
+        condition_list = ["As Is", "Acceptable", "Good", "Very Good", "Like New", "New"]
+        for i in range(len(condition_list)):
+            if i == self.condition:
+                return condition_list[i]
```

Comment:
Great work! I love this use of the list, just be mindful of the time complexity of this iteration. You could also use a conditional for each condition.